benchmark/misc/console.js

```diff
@@ -93,6 +93,8 @@ function runUsingArgumentsAndApply(n, concat) {
 function main(conf) {
   const n = +conf.n;
   switch (conf.method) {
+    // '' is a default case for tests
```

Comment:
Why are we always running the first case? Are the default cases missing a break? (Maybe I shouldn't comment if I'm missing context)

Reply:
@fhinkel We are limiting to the first case in tests only. `conf.method` is assigned a default value earlier in the code (line 16). Tests override it with a blank string by using `--set method=` as a command line option.

The reason is so that we don't have to change parameter names in all the benchmarks. 

Let's say `test-benchmark-misc` runs a benchmark called `foo.js`. It takes an configuration option called `method`. The `method` option may be `readFile` or `readFileSync`. If it's not one of those, the benchmark throws.

But `test-benchmark-misc` alos runs a benchmark called `bar.js`. It also takes a `method` option but throws if they are anything other than `chdir` or `chmod`.

It is now impossible to specify a configuration option (so that each benchmark file only runs a single benchmark) for `method` without causing one or the other to throw.

Rather than try to figure out unique names for each benchmark file (which seems like an anti-pattern--I don't want to have to know that `foo` uses `method` but `bar` uses `function` or something like that), we assign a default for the case where the configuration option is explicitly set to an empty value.

Reply:
That makes a lot of sense Thanks for the explanation!